.github/workflows/audit.yaml

```diff
@@ -31,7 +31,7 @@ jobs:
         python-version: ${{inputs.python_version}}
     - name: Install Python packages
       run: |
-        pip install --upgrade pip setuptools pytest coverage[toml]
+        python -m pip install --upgrade pip setuptools pytest coverage[toml]
```

Comment:
```suggestion
        python3 -m pip install --upgrade pip setuptools pytest coverage[toml]
```

Reply:
I recently learned that `python3` is not guaranteed to exist on Windows (while `python` is not guaranteed to exist on UNIX).

Reply:
curious...

Reply:
We could fix this but I *believe* the GitHub runners symlink python on unix for this reason. 
<img width="720" height="226" alt="image" src="https://github.com/user-attachments/assets/a4d09b31-90de-4132-8cd7-91437e0f3c63" />

Sigh.